SyncIntegrationTests/README.md

```diff
@@ -1,6 +1,6 @@
 # Firefox iOS integration tests
 
-To run these tests you will need [Python 2][] and [pipenv][] installed. Once
+To run these tests you will need [Python 3][] and [pipenv][] installed. Once
```

Comment:
The `Pipfile` declares that it requires python3, I wasn't able to follow the other instructions in this doc using python2.